org.envirocar.app/src/org/envirocar/app/views/recordingscreen/TrackMapFragment.java

```diff
@@ -330,7 +330,7 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
 
     @Override
     public void onExplanationNeeded(List<String> permissionsToExplain) {
-        Toast.makeText(getContext(), "We need access to your location.", Toast.LENGTH_SHORT).show();
+        showSnackbar("We need access to your location.");
```

Comment:
@SebaDro I am unsure about where to add these strings? should I create a new folder for the recording screen?

Reply:
I think _strings_notification.xml_ would be a good place for it.